### PR TITLE
Handle youtube url with a "s" at the end of the start time parameter.

### PR DESCRIPTION
**Pull request by UtherII**
Fix for bug reported at https://www.yaronet.com/topics/193011-video-youtube-avec-depart-decale#post-4

### CODE REVIEW

File: src/engine/media/widget.php [hunk 1]
```diff
@@ -341,7 +341,14 @@ function ($url, $code) {
     'embed.youtube' => array(
         '(?:www\\.youtube\\.com/watch\\?|youtu\\.be/([-0-9A-Za-z_]{1,64}))',
         function ($logger, $match, $query) {
-            $start = isset($query['t']) && is_numeric($query['t']) ? (int)$query['t'] : 0;
+            $start = 0;          
+            if (isset($query['t'])) {
+                $t_param = $query['t'];
+                if (strpos($t_param,"s")==strlen($t_param)-1){
+                    $t_param = substr($t_param,0,strlen($t_param)-1); 
+                }
```
Comment (r3c) [inline src/engine/media/widget.php]: [OPT] As an alternative you can leverage `substr`'s ability to deal with negative indices here:
```suggestion
                if (substr($t_param, -1) === 's') {
                    $t_param = substr($t_param, 0, -1); 
                }
```
Mind the triple "===" to perform a strict comparison, even though there should be no risk of nasty automatic cast here 🙂

**Review by r3c — DISMISSED** [review]
Thanks a lot! See inline comments for an alternative suggestion 🙂

**Review by r3c — APPROVED** [review]

**Comment by r3c** [comment]
Thanks again, feel free to merge whenever you please 🙂

**Comment by UtherII** [comment]
I'm not used to Github. I'm not sure how to merge. Are you sure I have the rights to do that myself? I can't see a merge button.

**Comment by r3c** [comment]
Oh sorry I think you do have the permission, unless I missed something in the configuration. You should have a "Rebase and merge" button appearing at the bottom of https://github.com/r3c/yaronet/pull/13 ; if you don't let me know, I'll merge the PR for you and will try to figure out what's wrong 🙂

**Comment by UtherII** [comment]
I guess there is actually an issue with permissions since at the bottom of the pull request page, I have : 

> This branch has no conflicts with the base branch
Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.

There are only a "Close pull request" button and a "Comment" button

**Comment by r3c** [comment]
OK then I just merged it, thanks again!

**Pull request merged**